bower.json

```diff
@@ -25,5 +25,8 @@
     "ember-cli-shims": "stefanpenner/ember-cli-shims#0.0.3",
     "jquery": "~2.1.1",
     "ember-data": "~1.0.0-beta.10"
+  },
+  "resolutions": {
+    "ember-data": "canary"
```

Comment:
This library is intended to support all known Ember versions.  As such, we need to test them all :open_mouth: :facepunch: :smiling_imp: